outline/getting-started.md

```diff
@@ -17,30 +16,24 @@ level tour of necessary knowledge for being productive with Tokio.
 A "Hello world!" Tokio tutorial. Implement a server that writes "hello world" to
 the socket.
 
-<a name="runtime-model"></a>
-## Runtime model
-
-**Status**: [Complete](https://tokio.rs/docs/getting-started/runtime-model/)
-
-A crash course into how Tokio executes code.
-
 <a name="futures"></a>
 ## Futures
 
-**Status**: [Complete](https://tokio.rs/docs/getting-started/futures/)
+**Status**: unassigned.
 
-A crash course into the Future abstraction and how Tokio uses it.
+A high level overview of how and when execution happens in Tokio, how futures
+are lazy and the basics of the polling model.
 
-<a name="tasks"></a>
-## Tasks
+<a name="errors"></a>
+## Error Handling
```

Comment:
Not sure if having basics of error handling here works, but might be nice.

Reply:
Yeah, could work! What would go in the section?

Reply:
I added some contents down below. I'm not 100% convinced this topic deserves its own section, but I'd need to see it in writing before being able to tell. Thoughts?

Reply:
I started playing with implementing this PR based on: https://github.com/tokio-rs/website/pull/223. So far I've simplified the Futures section and moved it front of the Runtime section. With some tweaking it's starting to feel good. I wonder if it's a better way to do things than what I've proposed here.